Geometry/Universes/rootUniverse_class.f90

```diff
@@ -84,12 +84,11 @@ subroutine init(self, fill, dict, cells, surfs, mats)
     character(nameLen)                                        :: name
     character(100), parameter :: Here = 'init (rootUniverse_class.f90)'
 
-    ! Set ID
-    call dict % get(id, 'id')
-    if (id <= 0) call fatalError(Here, 'Universe ID must be +ve. Is: '//numToChar(id))
-    call self % setId(id)
+    ! Setup the base class
+    ! With: id, origin rotations...
+    call self % setupBase(dict)
 
-    ! Make shure root does not contain neither origin nor rotation
+    ! Make sure root does not contain neither origin nor rotation
     if (dict % isPresent('origin')) then
```

Comment:
dict % isPresent('origin') 
is repeated twice in line 92 and 95

Reply:
Yikes!!! 
It has been there for long.... Good thing nobody was rotating root universes... :fearful:

Reply:
Should be resolved now.